integ-test/src/test/java/org/opensearch/sql/sql/DateTimeImplementationIT.java

```diff
@@ -5,18 +5,16 @@
 
 package org.opensearch.sql.sql;
 
-import org.junit.Test;
-import org.opensearch.sql.legacy.SQLIntegTestCase;
-
-import java.io.IOException;
-
 import static org.opensearch.sql.util.MatcherUtils.rows;
 import static org.opensearch.sql.util.MatcherUtils.schema;
 import static org.opensearch.sql.util.MatcherUtils.verifyDataRows;
 import static org.opensearch.sql.util.MatcherUtils.verifySchema;
 
-public class DateTimeImplementationIT extends SQLIntegTestCase {
+import java.io.IOException;
```

Comment:
Why changes from another PR are here?

Reply:
GJF covers order too, so later I can remove some of the non-Google Java Format tasks from the build.gradle